TW-965 changes for introduction page (#4608)
postmanlabs/postman-docs: src/pages/docs/getting-started/overview.md

    url: "https://blog.postman.com/postman-first-5-things-to-try/"
  - type: link
    name: "10 Postman Features Everyone Should Know"
    url: "https://blog.postman.com/10-postman-features-everyone-should-know/"
  - type: link
    name: "What Do 37,000 Developers Say About Postman? 3 Key Things"
    url: "https://blog.postman.com/what-developers-say-about-postman/"
  - type: subtitle
    name: "Public workspaces"
  - type: link
    name: "30 days of Postman for developers"
    url:  "https://www.postman.com/postman/workspace/30-days-of-postman-for-developers/overview"
---

Welcome! This section of the Learning Center contains topics about how to get started using Postman.

To get up and running with Postman, check out the following:

* You can get started right away by using the Postman web app, but you can also install the Postman desktop app. See [Installing and updating Postman](/docs/getting-started/installation-and-updates/).

* Once you're in Postman, check out how to [navigate Postman](/docs/getting-started/navigating-postman/). After you're familiar with the interface, try [sending your first request](/docs/getting-started/sending-the-first-request/).

> * Make sure to create a Postman account. After you have, learn more about [managing your account](/docs/getting-started/postman-account/), [customizing your Postman profile](/docs/getting-started/postman-profile/), and [setting up Postman](/docs/getting-started/settings/).

Reviewer: ```suggestion
* Create a Postman account. Then, learn more about [managing your account](/docs/getting-started/postman-account/), [customizing your Postman profile](/docs/getting-started/postman-profile/), and [setting up Postman](/docs/getting-started/settings/).
```

* A great place to find APIs is the Postman Public API Network. Learn more on how to use it at [exploring the Public API Network](/docs/getting-started/exploring-public-api-network/).

* After you've created a few requests, you'll want to organize them into collections and workspaces. Check out [creating your first collection](/docs/getting-started/creating-the-first-collection/) and [creating a workspace](/docs/getting-started/creating-your-first-workspace/).

* You can also import requests and other data into Postman, and export it for use with other programs. Check out [importing and exporting data](/docs/getting-started/importing-and-exporting-data/).

> * Having trouble? Check out how to [troubleshoot app issues](/docs/getting-started/troubleshooting-inapp/).

Reviewer: ```suggestion
* Having trouble? Learn how to [troubleshoot app issues](/docs/getting-started/troubleshooting-inapp/).
```
